src/cmdhandler/command_handler.cpp

```diff
@@ -86,6 +88,19 @@ std::vector<std::string> command_handler::tokenize_values_by_commas(const std::s
     return tokens;
 }
 
+cv::Mat command_handler::values_by_comma_to_mat(std::vector<std::string> values){
+    std::vector<double> values_as_double;
+    for(int i = 0; i < values.size(); i++){
+        try{
+            values_as_double.push_back(std::stod(values[i]));
+        }catch(const std::invalid_argument& err){
```

Comment:
Since you're not doing anything with the exception here, you should just let it bubble up to other try/catch blocks instead of catching and re-throwing the error. Just add a note to the function that an error can be thrown